Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -22,7 +22,7 @@ internal class WindowsMixedRealityDeviceManager : BaseDeviceManager
         /// <summary>
         /// Dictionary to capture all active controllers detected
         /// </summary>
-        private readonly Dictionary<uint, WindowsMixedRealityController> activeControllers = new Dictionary<uint, WindowsMixedRealityController>();
+        private readonly Dictionary<uint, IMixedRealityController> activeControllers = new Dictionary<uint, IMixedRealityController>();
```

Comment:
I thought we were going to change this to a list?

Reply:
I did , then looked back at my other code and found I had kept it.  It's useful for finding a controller by it's sourceID.

I simply changed the code that looped through them to use the Values array instead of the dictionary index